docs/static/ls-ls-config.asciidoc

```diff
@@ -1,98 +1,44 @@
-Use the Lumberjack protocol to connect two Logstash machines.
+* <<lumberjack-considerations,Lumberjack output to Beats input>>
+* <<http-considerations,HTTP output to HTTP input>>
 
-. Generate a trusted SSL certificate (required by the lumberjack protocol).
-. Copy the SSL certificate to the upstream Logstash machine.
-. Copy the SSL certificate and key to the downstream Logstash machine.
-. Set the upstream Logstash machine to use the Lumberjack output to send data.
-. Set the downstream Logstash machine to listen for incoming Lumberjack connections through the Beats input.
-. Test it.
+[[lumberjack-considerations]]*Lumberjack-Beats considerations*
 
-[[generate-self-signed-cert]]
-===== Generate a self-signed SSL certificate and key
+Lumberjack output to Beats input has been our standard approach for {ls}-to-{ls} communication, and may still be the best option for more robust use cases. 
+Before you implement the Lumberjack to Beats configuration, keep these points in mind: 
 
-Use the `openssl req` command to generate a self-signed certificate and key. The `openssl req` command is available with some operating systems. You may need to install the openssl command line program for others.
+* Lumberjack to Beats provides high availability, but does not provide load balancing. 
+The Lumberjack output plugin allows defining multiple output hosts for high availability, but instead of load-balancing between all output hosts, it falls back to one host on the list in the case of failure.
+* If you need a proxy between the Logstash instances, TCP proxy is the only option.
+* There's no explicit way to exert back pressure back to the beats input.
 
-Run the following command:
+Ready to see more configuration details? See <<ls-to-ls-lumberjack>>.
 
-[source,shell]
-----
-openssl req -x509 -batch -nodes -newkey rsa:2048 -keyout lumberjack.key -out lumberjack.cert -subj /CN=localhost
-----
+[[http-considerations]]*HTTP-HTTP considerations* 
 
-where:
+This approach relies on the use of <<plugins-outputs-http,http output>> to <<plugins-inputs-http,http input>> plugins.
+Take these considerations into account before you implement:
 
-* `lumberjack.key` is the name of the SSL key to be created
-* `lumberjack.cert` is the name of the SSL certificate to be created
-* `localhost` is the name of the upstream Logstash computer
+* HTTP does not provide built-in high availability. You will need to implement your own load balancer in between the HTTP output and the HTTP input. 
+* If you need a proxy between the Logstash instances, you can use any HTTP proxy. 
+* The HTTP input adds connection information to events, and this may be data you don't want.
 
+For now, <<plugins-outputs-http,http output>> to <<plugins-inputs-http,http input>> with manual configuration may be the best path forward if these limitations don't apply to your use case.
 
-This command produces output similar to the following:
+// Ready to see more configuration details? See <<ls-to-ls-http>>.
```

Comment:
Notice that I have Line 39 in the ls-ls-config.asciidoc` file commented out. 

The file is present and linked, it appears in the right nav, and users can navigate to it. When we flesh out http config details, we should only need to uncomment this line to activate the link. 

In the meantime, I don't want to tease users with more config details that we're still working on.